.storybook/main.js

```diff
@@ -23,5 +23,6 @@ const config = {
     autodocs: "tag",
     toc: true,
   },
+  staticDirs: ["../public"],
```

Comment:
Is there any way of having this static folder within the `/elements/storytelling/stories/` folder? It would make more sense having it there. If not, maybe we can introduce a structure like `/public/stories/storytelling/sample.md`.

Reply:
Yeah we can rather than passing `["../public"]` we can pass `["../elements/storytelling/stories/public/"]`